docs/source/testing/larva/ignores.rst

```diff
@@ -0,0 +1,284 @@
+
+.. code-block::
+
+  ignoreKeysAndContentBetweenKeys1.key1=<RecordID>
+  ignoreKeysAndContentBetweenKeys1.key2=</RecordID>
+
+
+**ignoreContentBeforeKey**
+
+This replaces everything before provided key has been found with IGNORE. By writing
+
+.. code-block::
+
+  ignoreContentBeforeKey1.key=ABC
+
+content before “ABC”, in this case is “<RecordID> 12</RecordID> <Category>”, will be replaced with IGNORE.
+
+**ignoreContentAfterKey**
+
+The opposite of ignoreContentBeforeKey is ignoreContentAfterKey, this replaces everything after  provided key has been found with IGNORE. By writing 
+
+.. code-block::
+
+  ignoreContentAfterKey1.key=ABC
+
+it will replace everything after “ABC” with IGNORE.
+
+**ignoreCurrentTimeBetweenKeys**
+
+This replaces time found between key1 and key2 with IGNORE_CURRENT_TIME, pattern can be null. Write as
+
+.. code-block::
+
+  ignoreCurrentTimeBetweenKeys1.key1=<Timestamp>
+  ignoreCurrentTimeBetweenKeys1.key2=</Timestamp>
+  ignoreCurrentTimeBetweenKeys1.pattern=yyyy-MM-dd'T'HH:mm:ss.SSSZ
+  ignoreCurrentTimeBetweenKeys1.margin=12345
+  ignoreCurrentTimeBetweenKeys1.errorMessageOnRemainingString=true/false
+
+
+**Other way of using ignore**
+
+Since IAF 7.6 version, it allows ignoring the result of a specific test step by writing ``step.service.read=ignore``. For example, in the ``scenario.properties`` file, you write ``step7.database.Generic.read=ignore`` to ignore the result of this step.
+
+
+Using replace
+-----------------
+
+**replaceRegularExpressionKeys**
+
+This replaces all matches of regular expression (key1) with the value of key2. By writing
+
+.. code-block::
+
+  replaceRegularExpressionKeys1.key1=RecordID
+  replaceRegularExpressionKeys1.key2=ID
+
+all 4 “RecordID” will be replaced by “ID”.
+
+**replaceKey / replaceEverywhereKey**
+
+replaceKey and replaceEverywhereKey work the same, both replace all matching occurrences of provided key with provided value, write as:
+
+.. code-block::
+
+  replaceKey1.key1=RecordID
+  replaceKey1.key2=ID
+
+and
+
+.. code-block::
+
+  replaceEverywhereKey1.key1=RecordID
+  replaceEverywhereKey1.key2=ID
+
+
+Using remove
+-----------------
+
+**removeRegularExpressionKey**
+
+This replaces all matches of regular expression with “”, write as
+
+.. code-block::
+
+  removeRegularExpressionKey1.key=\\d
+
+all digits will be removed from the result.
+
+**removeKeysAndContentBetweenKeys**
+
+This removes all characters found between key1 and key2, including key1 and key2. By writing
+
+.. code-block::
+
+  removeKeysAndContentBetweenKeys1.key1=<RecordID>
+  removeKeysAndContentBetweenKeys1.key2=</RecordID>
+
+“<RecordID> 12</RecordID>” will be removed from the result.
+
+**removeKey**
+
+This removes all matching occurrences of provided key, by writing
+
+.. code-block::
+
+  removeKey1.key=RecordID
+
+all 4 “RecordID” will be removed from the result.
+
+
+Others
+-----------------------
+
+**decodeUnzipContentBetweenKeys**
+
```

Comment:
If you process the text &lt;File&gt;myFile&lt;/File&gt; and if you use decodeUnzipContentBetweenKeys, does Larva try to read a file with name "myFile", or is "myFile" interpreted as zipped data that has to be unzipped?